src/components/Toggle/Toggle.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import styles from "./Toggle.module.scss";
+import { Col, Row, Switch, Typography } from "antd";
+import classNames from "classnames";
+
+interface ToggleProps {
+  checked?: boolean;
+  onChange?: (i: boolean) => void;
+  label?: string;
```

Comment:
`label` is not assigned a default value. So, it should be a required prop.

Reply:
Label is being used for conditional rendering, therefore it is set as optional

Reply:
Best, Thank you.